modules/claims_api/lib/claims_api/claim_logger.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  # Logger for ClaimsApi, mostly used to format a print statement
+  # Uses the standard Rails logger
+  #
+  #   ClaimsApi::Logger.log('526ez', claim_id: 'id_here')
+  #   # => "ClaimsApi :: 526ez :: Claim ID: id_here"
+  class Logger
+    LEVELS = %i[debug info warn error fatal unknown].freeze
+
+    def self.log(tag, **params)
+      level = pick_level(**params)
+      params.delete(:level)
+      msg = format_msg(tag, **params)
+
+      Rails.logger.send(level, msg)
+      msg
+    end
+
+    def self.pick_level(**params)
+      params.key?(:level) && params[:level].to_sym.in?(LEVELS) ? params[:level].to_sym : :info
+    end
+
+    def self.format_msg(tag, **params)
+      msg = ['ClaimsApi', tag]
+      case tag
+      when '526'
+        msg.append("Claim ID: #{params[:claim_id]}") if params[:claim_id].present?
+        msg.append("Detail: #{params[:detail]}") if params[:detail].present?
+        msg.append("VBMS ID: #{params[:vbms_id]}") if params[:vbms_id].present?
+        msg.append("autoCestPDFGenerationDisabled: #{params[:pdf_gen_dis]}") if params[:pdf_gen_dis].present?
+        msg.append("Attachment ID: #{params[:attachment_id]}") if params[:attachment_id].present?
+      when 'poa'
+        msg.append("PoA ID: #{params[:poa_id]}") if params[:poa_id].present?
```

Comment:
```suggestion
        msg.append("POA ID: #{params[:poa_id]}") if params[:poa_id].present?
```
nitpick for consistency: it should always be styled as POA